### ISSUE TITLE
MADE masking like tfb.AutoregressiveNetwork

### ISSUE THREAD

**Issue by jecampagne** [deepmind/distrax#204]
Hello,
Am I right to say that SplitCoupling seems related to the Real NVP simple coupling, and MaskCoupling can implement a chessboard coupling?
And what about the autoregressive MADE like masking strategy, is it implemented somewhere?
I am looking for an equivalent of TF bijectors
```python
        hidden_units = [128, 128]
        bijectors.append(tfb.MaskedAutoregressiveFlow( name ='MAF%d' %i, 
            shift_and_log_scale_fn=tfb.AutoregressiveNetwork(
                params=2, hidden_units=hidden_units, activation=tf.nn.leaky_relu)))

```
Thanks
